docs/src/main/sphinx/connector/ignite.rst

```diff
@@ -0,0 +1,203 @@
+================
+Ignite connector
+================
+
+.. raw:: html
+
+  <img src="../_static/img/ignite.png" class="connector-logo">
+
+The Ignite connector allows querying an `Apache Ignite <https://ignite.apache.org/>`_
+database from Trino.
+
+Requirements
+------------
+
+To connect to a Ignite server, you need:
+
+* Ignite version 2.8.0 or latter
+* Network access from the Trino coordinator and workers to the Ignite
+  server. Port 10800 is the default port.
+
+To enable Ignite Connector, you need:
+* Specify ``--add-opens=java.base/java.nio=ALL-UNNAMED`` in the ``jvm.config`` when starting the Trino server.
+
+Configuration
+-------------
+
+The Ignite connector default has two schemas: ``sys`` and ``public``. The ``sys``
+schema, which contains a number of system views with information about cluster nodes,
+user can not create or modify any data among this schema. The ``public`` schema,
+which is used by default whenever a schema is not specified.
+
+The connector can query a Ignite instance. Create a catalog properties file
```

Comment:
Can you update this to use an approach like at https://trino.io/docs/current/connector/postgresql.html#configuration

Reply:
I think you missed this update @chenjian2664 

it should talk about example.properties and so on